### ISSUE TITLE
Show 'discovery not completed' in one ADLS Gen2 blob deletion log message 

### ISSUE THREAD

**Issue by v-xianya** [microsoft/AzureStorageExplorer#2195]
**Storage Explorer Version:** 1.11.0
**Build:** [20191101.5](https://devdiv.visualstudio.com/DevDiv/_build/results?buildId=3203746)
**Branch:** rel/1.11.0
**Platform/OS:** Windows 10/ Linux Ubuntu 19.04/macOS High Sierra
**Architecture:** ia32/x64
**Regression From:** Not a regression

**Steps to reproduce:**
1. Expand one ADLS Gen2 storage account -> Blob Containers.
2. Select one blob container and upload a blob.
3. Delete the uploaded blob.
4. Observe the log message on Activities panel.

**Expect Experience:**
1. Delete successfully.
2. Show 'discovery completed' in log message.

**Actual Experience:**
1. Delete successfully.
2. Show 'discovery not completed'.
![image](https://user-images.githubusercontent.com/41351993/68021504-8839ba00-fcdc-11e9-8d88-d1efe912bbfa.png)

**More Info:**
For one regular blob, the log message shows like below.
![image](https://user-images.githubusercontent.com/41351993/68021801-6987f300-fcdd-11e9-9d28-c933c0e0112b.png)

**Comment by v-xianya** [comment]
Hi @JasonYeMSFT 
Verified this issue on build 20191102.1. We found the deletion log message shows like below:

1. For one ADLS Gen2 blob. '**discovery not completed**' has been removed from deletion log message .
![image](https://user-images.githubusercontent.com/41351993/68097329-72e9a900-fef1-11e9-87cb-d1861ea607c6.png)

2. For one regular blob, '**discovery completed**' still shows well in deletion log message .
![image](https://user-images.githubusercontent.com/41351993/68101558-2b6f1700-ff09-11e9-9278-8632b6ff507f.png)

    **Is this expected?**
